Makefile

```
@@ -407,6 +407,9 @@ test-e2e-local-migration: $(GINKGO)
 test-e2e-local: $(GINKGO)
 	./hack/test-e2e-local.sh --procs=$(PARALLEL_E2E_TESTS) --label-filter="default"
 
+test-e2e-local-ha: $(GINKGO)
+	./hack/test-e2e-local.sh --procs=$(PARALLEL_E2E_TESTS) --label-filter "default || (high-availability && $(HA_MODE))"
```

After https://github.com/gardener/gardener/pull/6914, this will be called 

```suggestion
	./hack/test-e2e-local.sh --procs=$(PARALLEL_E2E_TESTS) --label-filter "default || (high-availability && $(SHOOT_FAILURE_TOLERANCE_TYPE))"
```